client/app/components/pages.js

```diff
@@ -10,7 +10,8 @@ import topbar from "./_topbar/topbar";
 import landingPage from "./landingPage/landingPage";
 import notifications from "./notifications/notifications";
 import groupInfo from "./groupInfo/groupInfo";
-import groupList from "./groupList/groupList";
+import groupList from "./_groupList/groupList";
```

Comment:
Yeah, makes sense.
I thought I might include it on a userPage soon, so that you can see what groups a user is part of. Do you think that's a good idea? Will change it otherwise